### PR TITLE
Added New Demos + Minor Changes

### PR DESCRIPTION
**Pull request by AliyanH**
- Added data directory to store mapml data (ex. OSM basemap)
- Added 4 new demos:
   - **map-feature** - Added example from @ahmadayubi
   - **map-link** - Added simple example highlighting the attribution link
   - **map-a** - Made a simple example of Ontario linking to Ottawa and Ottawa linking to the website of Ottawa
   - **map-meta** - Added example highlighting 'projection' and 'zoom'

### CODE REVIEW

File: static/demo/layer-demo.html [hunk 1]
```diff
@@ -80,8 +80,9 @@
 </head>
 <body>
     <div style="text-align: center;">
-        <h2>MapML Demo: &lt;layer-&gt;</h2>
-        <button class="run" onclick="plot()">▶</button>
+      <!------------------------------------------------- CHANGE THIS ------------------------------------------------->
+      <h2>MapML Demo: &lt;layer-&gt;</h2>
+      <button class="run" onclick="plot()">▶</button>
```
Comment (Malvoz) [inline static/demo/layer-demo.html:85]: I haven't tested this using a screen reader: `<button class="run" onclick="plot()">▶</button>`, is it announced correctly?

https://scottvinkle.me/blogs/work/svg-and-emoji-icon-accessibility#accessible-emoji:

> Take for instance the common phrases, "I ❤️ you" – announced as, "I red heart you."

Perhaps these buttons should be changed to:

```diff
<button class="run" onclick="plot()"
+ aria-label="run">
- ▶
+ <span aria-hidden="true">▶</span>
</button>
```

File: static/demo/map-a-demo.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
... -79.2984996 46.9207191 -79.2987816 46.9209288 -79.2991338 46.9211846 -79.2995155 46.9214975 -79.2999274 46.9217645 -79.300362 46.9220429 -79.30075 46.9222792 -79.3010715 46.9225274 -79.30146 46.9227485 -79.3018564 46.9229621 -79.302192 46.9231948 -79.302384 46.9233208 -79.3025902 46.923504 -79.3029246 46.9236258 -79.3031993 46.923828 -79.3035735 46.9240608 -79.3040079 46.9242896 -79.3044358 46.9245224 -79.3049158 46.9248008 -79.3053516 46.9250031 -79.305718 46.9251977 -79.3061144 46.9254112 -79.3064878 46.9256782 -79.3069149 46.9259452 -79.3073501 46.9261856 -79.3077385 46.92638 -79.3080369 46.9265633 -79.3082586 46.9267045 -79.308685 46.9269142 -79.3090436 46.9270935 -79.3094179 46.9273033 -79.3098979 46.9275817 -79.310349 46.9279023 -79.3106691 46.9281502 -79.3110126 46.9283982 -79.3113091 46.9286917 -79.3116072 46.9291 -79.3118897 46.9293862 -79.3122178 46.9297218 -79.3124843 46.9300422 -79.3126911 46.9302789 -79.3127447 46.9303247 -79.3131864 46.9307441 -79.3133767 46.9309615 -79.3136449 46.9312821 -79.3139191 46.931591 -79.3141477 46.9318275 -79.3144985 46.9321746 -79.3148263 46.9325065 -79.3151696 46.9327964 -79.3155449 46.9330903 -79.3159791 46.9334755 -79.3163454 46.933921
```
Comment (prushforth) [inline static/demo/map-a-demo.html:53]: This link is 404.

Comment (prushforth) [reply static/demo/map-a-demo.html:53]: This was a LHIN boundary I think.  So the link should go here: https://www.ottawapublichealth.ca/

Comment (prushforth) [reply static/demo/map-a-demo.html:53]: But maybe we should think about making the demo go from Canada to the provinces layer i.e. I think the link from Ontario to Ottawa is good, but would make more sense from Ontario to whatever breakdown of Ontario we were mapping e.g. the LHIN boundaries or Census Divisions, perhaps.  The problem with *that* is the volume of data we have to include in the demo player.  Needs a bit more design thought.

Comment (prushforth) [reply static/demo/map-a-demo.html:53]: Even a polygon to a set of point markers would be reasonable

File: static/demo/map-meta-demo.html [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+        <!------------------------------------------------- CHANGE THIS ------------------------------------------------->
+        <textarea rows="10" style="width:100%" id="htmlEditor">
+<mapml-viewer projection="OSMTILE" zoom="3" lat="45.41868" lon="-75.690276" controls>
+  <layer- label="OpenStreetMap" src="data/osm.mapml" checked></layer->
+  <layer- label="Favourite Restaurant" checked>
+    <map-meta name="projection" content="OSMTILE"></map-meta>
+    <map-meta name="zoom" content="min=3,max=10,value=3"></map-meta>
+    <map-feature class="favourite">
+      <map-featurecaption>Big Daddy's Crab Shack</map-featurecaption>
+      <map-geometry cs="gcrs">
```
[the remark this answers is not in the document]
Comment (AliyanH) [reply static/demo/map-meta-demo.html]: updated! 14bfdaf352d249c2a7ce6b343255fc24c1b89971

File: static/demo/map-a-demo.html (hunk 2 above)
Comment (AliyanH) [reply static/demo/map-a-demo.html]: Updated the demo's for map-a and map-feature, please comment if any changes need to be made! Thanks!

**Pull request merged**